### ISSUE TITLE
Font Ligatures Not Working on XCode

### ISSUE THREAD

**Issue by jsanzdev** [githubnext/monaspace#23]
The font ligatures are not working on XCode, if I use JetBrains Mono the ligatures work perfectly.

**Comment by maniel** [comment]
yeah, i found those aren't working everywhere, for example in [Notepad++](https://github.com/notepad-plus-plus/notepad-plus-plus), [Windows Terminal](https://github.com/microsoft/terminal) or [Notepads](https://github.com/0x7c13/Notepads) don't show ligatures for me in any of Monaspace fonts but shows them with Fira Code font

**Comment by trispo** [comment]
When I enable ligatures in Nova.app or Warp.app, they work fine. But not in Xcode.

**Comment by smolinari** [comment]
They don't seem to work in VSCode on Linux either. 

Scott

**Comment by joelbutcher** [comment]
Having issues with them in Jetbrains IDE's as well

**Comment by coffius** [comment]
> Having issues with them in Jetbrains IDE's as well
The same 😔
Ligatures do not work in `IntelliJ IDEA 2023.1 (Community Edition)`
<img width="745" alt="image" src="https://github.com/githubnext/monaspace/assets/2510890/74387c3a-6be4-4967-a76c-571956b07ce5">

**Comment by ppshobi** [comment]
same here, phpstorm latest, ligatures does not render

**Comment by ppshobi** [comment]
https://github.com/githubnext/monaspace/issues/13

**Comment by humblehacker** [comment]
Normally you'd enable/disable font features like this from the typography popup in the font picker, but unfortunately Xcode seems to ignore these settings.

<img width="996" alt="Screenshot 2023-11-10 at 9 44 18 AM" src="https://github.com/githubnext/monaspace/assets/117582/f6080ede-4b4c-4d69-97e0-b801639c68f4">

**Comment by NathanHannon** [comment]
I don't see why they can't all just be enabled by default like what other fonts with ligatures have.

**Comment by oronbz** [comment]
Same here, ligatures not working on all monaspace fonts in Xcode.

**Comment by sebastianmarkow** [comment]
Terminal.app as well.

**Comment by trispo** [comment]
@oronbz They work with Fira Code in Xcode.

**Comment by oronbz** [comment]
> @oronbz They work with Fira Code in Xcode.

I know, never had any complaint for Fira Code :D

**Comment by crsantos** [comment]
~~Xcode~~ Xcode

**Comment by atomicbird** [comment]
Ligatures work in TextEdit on macOS, so I wonder if this is Apple's bug instead of Monaspace.

**Comment by yam-liu** [comment]
Same here, any progress?

**Comment by ravero** [comment]
Same here...

**Comment by heathercran** [comment]
Programs that don't allow enabling of stylistic sets are unable to show Monaspace's ligatures currently. The next update will include a "frozen" version of the fonts with all ligatures enabled by default to bypass this limitation.

**Comment by sgigou** [comment]
Just to give another point of view on this issue: that "bug" is actually one of the things that please me in Monaspace.

As you cannot choose what set to use in those apps, I prefer to have none than all of them. I cannot work with ligatures from ss01 and ss02 sets… The default enabling actually made me quit Berkeley Mono and Fira Code already.

**Comment by heathercran** [comment]
@sgigou that's totally fair! the non-frozen version of the fonts will still be available as the default download; this alternate option is just for those using programs that don't support stylistic sets.

**Comment by idan** [comment]
This is resolved in 1.2 to the best of our abilities within the limitations. XCode does not support setting of opentype features like stylistic sets, and hence we produce the new "Frozen" font builds in 1.2 to support your use case.
